packages/docusaurus-theme-classic/src/theme/DocSidebarItem/styles.module.css

```diff
@@ -6,18 +6,6 @@
  */
 
 @media (min-width: 997px) {
-  .menuLinkText {
-    cursor: initial;
-  }
-
-  .menuLinkText:hover {
-    background: none;
```

Comment:
Also, we don't need this CSS class after introducing `menu__list-item-collapsible`.